### ISSUE TITLE
Accessibility audit: change footer headings from h3s to h2s

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#3922]
e.g "Get help", "About PyPI", etc.

> Footer headings are marked as h3 elements, which is a bit confusing. Using the landmarks view or navigating between headers using a shortkey reveals that the footer headings are
read as subheadings to the "New releases" heading. These headings should either be h2 elements as
well or not be headings at all.

---

**Good First Issue**: This issue is good for first time contributors. If you've already contributed to Warehouse, please work on [another issue without this label](https://github.com/pypa/warehouse/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22good+first+issue%22) instead. If there is not a corresponding pull request for this issue, it is up for grabs. For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/). If you are working on this issue and have questions, please feel free to ask them here, [`#pypa-dev` on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or the [pypa-dev mailing list](https://groups.google.com/forum/#!forum/pypa-dev).

**Comment by jaystino** [comment]
I've gone ahead and changed the four footer headings--Get help, About PyPI, Contributing to PyPI, Using PyPI-- from h3 elements to h2 elements in `base.html`. I checked the changes in the browser and ran the tests, which all appeared to have passed. Is there anything else you'd like me to do before submitting a pull request?

**Comment by nlhkabu** [comment]
Thanks @jaystino :)
You may also need to update the CSS here: https://github.com/pypa/warehouse/blob/master/warehouse/static/sass/blocks/_footer.scss#L87

Once this is done, please make a PR - feel free to ping me so I can review it :)

**Comment by jaystino** [comment]
Hi Nicole, thanks for helping me out with my first contribution!

Just in case tagging you in the pull request comment doesn't ping you for
some reason, I just wanted to let you know that I made the additional
change to the .css file and submitted the pull request.

Thanks again :)
Jay

Jay Ottenstein
+1 240 425 7803
+46 722 98 13 95
LinkedIn <http://www.linkedin.com/in/jayot>
